docs/usage.rst

```diff
@@ -734,11 +734,13 @@ The following examples show how these methods map to each other.
     >>> selector.css('a::attr(href)')[0].getall()
     ['image1.html']
 
-So, the main difference is that output of ``.get()`` and ``.getall()`` methods
-is more predictable: ``.get()`` always returns a single result, ``.getall()``
-always returns a list of all extracted results. With ``.extract()`` method
-it was not always obvious if a result is a list or not; to get a single
-result either ``.extract()`` or ``.extract_first()`` should be called.
+With the ``.extract()`` method it was not always obvious if a result is a list
+or not; to get a single result either ``.extract()`` or ``.extract_first()``
+needed to be called, depending whether you had a `Selector` or `SelectorList`.
```

Comment:
would you mind changing it to double `` quotes? Single make text italic in .rst files.

Reply:
I thought italics looked better to not distract from the `extract` & `extract_first` methods but I think you're right, it's better like this